containers/org.eclipse.linuxtools.docker.core/src/org/eclipse/linuxtools/internal/docker/core/DockerClientFactory.java

```diff
@@ -104,6 +104,12 @@ public DockerClient getClient(
 			builder.registryAuthSupplier(
 					new FixedRegistryAuthSupplier(registryAuth, configs));
 		}
+		// TODO This is giant hack to make jersey ClientBuilder initialized
+		// Should be removed before merging
```

Comment:
FWIW: I added a commit so the ClientBuilder property setting is put back after the build.  I tried the same for the RuntimeDelegate but it failed when the connection got opened.  It will probably require setting/restoring the property in another location.